core/src/lib.rs

```diff
@@ -38,11 +38,14 @@ pub struct EVMAddress(#[serde(with = "hex::serde")] pub [u8; 20]);
 /// Type alias for withdrawal payment, HashType is taproot script hash
 pub type WithdrawalPayment = (Txid, HashType);
 
-#[derive(Clone, Debug, Eq, Hash, PartialEq, PartialOrd, Ord, Serialize, Deserialize)]
+#[derive(
+    Clone, Debug, Eq, Hash, PartialEq, PartialOrd, Ord, Serialize, Deserialize, sqlx::FromRow,
```

Comment:
FromRow seems inappropriate here since it is a column type

Reply:
Done